### PR TITLE
Created Codingbat2 Post

### PR DESCRIPTION
**Pull request by ghost**
http://jacobthill.github.io/spring2014/

### CODE REVIEW

**Comment by gerbal** [comment]
@jacobthill check the filename of this post

File: _posts/2014-02-05-testytest.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (eah13) [inline _posts/2014-02-05-testytest.md:1]: @jacobthill this file needs to be renamed or deleted- it's a conflict with an existing filename.

**Comment by eah13** [comment]
@jacobthill nice work!

Just rename that file (from Github would be fine) and you're good to go.

**Comment by ghost** [comment]
Hi Grant,

I'm not sure what you are referring to in this email above. I also tried
the things you and Elliott suggested before emailing and it didn't seem to
work. Do you have any time that you can meet on Monday or Tuesday to help
troubleshoot these issue? I don't think it will take long, but I would like
to be caught up before class on Wednesday.

Thanks,
Jacob

**Comment by gerbal** [comment]
@jacobthill sorry, I think I posted a comment in the wrong pull request. 

I meant to link to your files in this pull request, a couple of them have conflicts that need to be resolved. I.e. rename the files so they don't cause conflicts with other files of the same name. 

Also, revert the changes to `main.css`

and last you need to encolse your [code block in jacob_simpleapp](https://github.com/silshack/spring2014/pull/262/files#diff-1c6e1d7445a068c2534229e56514def2R19) with `{% raw %}` `{% endraw %}` so jekyll won't try to interpret the liquid-esque code from simple.

**Comment by gerbal** [comment]
I'm also available monday afternoon, and tuesday for a few hours, and then all tuesday evening.

**Comment by ghost** [comment]
Ok, I will try to figure those things out. Can we meet for a few minutes tomorrow afternoon? I get out of class at 12:00, so whenever you are available after that.

Thanks

**Comment by gerbal** [comment]
I’ll be in the SILS library from 12:30 to 5

**Comment by eah13** [comment]
@jacobthill quite a few problems with this pull request.  Seems like you've deleted several other peoples' posts.  We'll need to restore those, likely manually at this point.  ~~Also, your pipes and filters should be off of a new branch- see the exercise from last class if you're unsure of how to do this.~~ Edit: just saw your new PR- disregard that comment.  Still need to get this repo cleaned up.

Happy to schedule a time to chat via video if that's helpful.

**Pull request closed without merging**